classes/class_06/CodeSamples/var/ConstStudent.java

```diff
@@ -0,0 +1,23 @@
+package var;
+
+public  class ConstStudent {
+        int rollNo; 
+        String studentName; 
+		
+        ConstStudent (){
+            this(10,"Sam");
+         }    
+        ConstStudent( int rollNo, String studentName){ 
+                   super();

```

Comment:
We don't really need to call `super()` here. Might be more confusing.

Reply:
to the extent needed to understand the concept, i will explain them in the class.